yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -1301,7 +1330,13 @@ def pixelize_sph_kernel_slice(
                         continue
 
                     # see equations 6, 9, and 11 of the SPLASH paper
-                    buff[xi, yi] += prefactor_j * kernel_func(q_ij)
+                    local_buff[xi + yi*xsize] += prefactor_j * kernel_func(q_ij)
+
+        with gil:
```

Comment:
as long as this is the recommended way to do the reduce, i think it's okay, but i had to check indentation levels to make sure I understood what was happening and which scope this was in.

Reply:
I don't know if it's **the** recommended, but that was the only one that worked :)